### PR TITLE
locator changes as per latest sandbox UI

### PR DESCRIPTION
**Pull request by sawantpritam**
@itsankit-google @rmstar Please review.

### CODE REVIEW

File: src/main/java/io/cdap/e2e/utils/ElementHelper.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public static void clickOnElementUsingJsExecutor(WebElement element) {
    */
   public static void clickOnElement(WebElement element) {
     WaitHelper.waitForElementToBeClickable(element);
-    scrollToElement(element);
+    scrollToElementUsingJsExecutor(element);
```
Comment (itsankit-google) [inline src/main/java/io/cdap/e2e/utils/ElementHelper.java]: Why is this change needed? 
Using js executor does not simulate user actions.

Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/utils/ElementHelper.java]: Normal `scrollToElement(element)` is not working in new left side panel.

image

Comment (itsankit-google) [reply src/main/java/io/cdap/e2e/utils/ElementHelper.java]: This change might be related to the recent migration of Pipeline Studio from angular to react.
But we cannot make this action use JS Executor since this is being used across all tests.
Please check with @mrahanjam & @sumengwang.

Comment (sumengwang) [reply src/main/java/io/cdap/e2e/utils/ElementHelper.java]: We want to fix the ui to match the e2e tests as much as possible. It will be good to just identify the issues first. But I believe this pr is necessary.

**Review by sumengwang — DISMISSED** [review]
LGTM

File: src/main/java/io/cdap/e2e/utils/ElementHelper.java (hunk 1 above)
Comment (itsankit-google) [reply src/main/java/io/cdap/e2e/utils/ElementHelper.java]: Can we please open a JIRA to track this? I am fine with this change if it is temporary but in longer term using JS Executor might not be the correct way to interact with UI elements in end-to-end tests.

Comment (sumengwang) [reply src/main/java/io/cdap/e2e/utils/ElementHelper.java]: https://github.com/cdapio/cdap-ui/pull/605/files
this should fix the scroll issue. The footer will no longer block the elements. no need for jsexecutor

Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/utils/ElementHelper.java]: Removed use of JS Executor. Above change to fix scroll issue is not available in latest sandbox successful build. @itsankit-google Please re-trigger e2e tests after next successful sandbox build.

**Review by itsankit-google — COMMENTED** [review]
Please look into test failures.

**Review by itsankit-google — APPROVED** [review]
